docs/resources/rule_amqp.md

```diff
@@ -0,0 +1,63 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "ably_rule_amqp Resource - terraform-provider-ably"
+subcategory: ""
+description: |-
+  The ably_rule_amql resource allows you to create and manage an Ably integration rule for AMQP. Read more at https://ably.com/docs/general/firehose/amqp-rule
```

Comment:
Should be ably_rule_amqp

Reply:
Whoops. Must have typod a search and replace.